rocAL/rocAL/source/api/rocal_api_meta_data.cpp

```diff
@@ -215,83 +216,38 @@ ROCAL_API_CALL rocalGetImageId(RocalContext p_context,  int* buf)
     }
 }
 
-void
-ROCAL_API_CALL rocalGetImageLabels(RocalContext p_context, void* buf, unsigned int flags)
+RocalTensorList
+ROCAL_API_CALL rocalGetImageLabels(RocalContext p_context)
```

Comment:
I think GetImageLables has been changed for image pipeline. Please double check if we need to apply the same here for performance reasons